cmd/install/addon/cmd.go

```diff
@@ -242,8 +268,51 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	billingModel := args.billingModel
+	billingModelAccountID := args.billingModelAccountID
+
+	if !cmd.Flags().Changed(billingModelFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && !cmd.Flags().Changed(billingModelFlag) {
+		billingModel, err = interactive.GetString(interactive.Input{
+			Question: "Billing Model",
+			Help:     cmd.Flags().Lookup(billingModelFlag).Usage,
+			Default:  "standard",
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid billing model: %s", err)
+			os.Exit(1)
+		}
+	}
+
+	if !cmd.Flags().Changed(billingModelAccountIDFlag) && !interactive.Enabled() {
+		interactive.Enable()
+		r.Reporter.Infof("Enabling interactive mode")
+	}
+
+	if interactive.Enabled() && billingModel != string(v1.BillingModelStandard) &&
+		!cmd.Flags().Changed(billingModelAccountIDFlag) {
+		billingModelAccountID, err = interactive.GetString(interactive.Input{
+			Question: "Billing Account ID",
+			Help:     cmd.Flags().Lookup(billingModelAccountIDFlag).Usage,
+			Required: true,
+		})
```

Comment:
should we default to the user's current aws account? does it even make sense to ask for this in the ROSA context when there is already an AWS account defined?

Reply:
no because that is the current behavior, we inherit the billing model and struct from the cluster. But this is to allow addons be purchased from different accounts. So we can not assume the users aws account.